app/src/main/java/com/gholem/moneylab/features/add/BottomSheetCategoryFragment.kt

```diff
@@ -1,25 +1,34 @@
 package com.gholem.moneylab.features.add
```

Comment:
Please add a new package inside `add` package i.e. `chooseTransactionCategory` and move all classes connected to BottomSheetCategoryFragment there (of course add another packages inside where needed - like `viewmodel` )